doc/users/install.rst

```diff
@@ -119,7 +125,7 @@ Must Have
 Nibabel_ 1.0 - 1.4
```

Comment:
i agree. perhaps we can simply replace it with a link to the requirements file.

it would be good to simplify this to:

1. install with pip
2. if you don't have sudo privileges, 
    a. install a miniconda environment 
    b. install nipype into it.
3. if you don't want to do either talk to your sysadmin :)